components/navbar.css

```diff
@@ -0,0 +1,40 @@
+.nav-header{
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 1rem 8px;
+    box-shadow: 0 2px 5px 0 var(--box-shadow), 0 2px 10px 0 var(--box-shadow);
+}
+
+.nav-items{
+    margin-left: auto;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.nav-right{
+    margin-right: auto;
```

Comment:
`.nav-right` items should have a `margin-left:auto` isn't?
or is the naming of class wrong here?

Reply:
This class is for moving the nav items to right corner. Should I change the name of the class?